devtools-ci-index.yaml

```diff
@@ -447,6 +447,42 @@
             cico node done $CICO_ssid
             exit $rtn_code
 
+- job:
```

Comment:
@ldimaggi Do you still need this job ? It has passed long time since you opened the PR.